ansible/roles/python/templates/etc/pip.conf.j2

```diff
@@ -0,0 +1,7 @@
+# {{ ansible_managed }}
+
+# Copyright (C) 2020 Robin Schneider <ypid@riseup.net>
+# SPDX-License-Identifier: GPL-3.0-only
```

Comment:
When possible, the copyright notice should be for the template and not the final config - in other words, put it in `{# #}` jinja comment section. Exceptions to this are bash/python scripts since they need to pass `shellcheck` and `pycodestyle` checks.

Reply:
Makes sense. Just for this reason I checked some other recently added role to follow the same style. Seems I picked the wrong role model file. Updated both.